vesc_driver/src/vesc_interface.cpp

```diff
@@ -221,21 +243,21 @@ void VescInterface::disconnect()
     impl_->rx_thread_run_ = false;
     impl_->rx_thread_->join();
 
-    std::lock_guard<std::mutex> lock(impl_->serial_mutex_);
+    // std::lock_guard<std::mutex> lock(impl_->serial_mutex_);
```

Comment:
Same.

Reply:
i just ran a test to check and in case the port is wrong the error is

`[vesc_driver]: Failed to connect to the VESC, SerialException Failed to open the serial port / tmp/ttyV0 to the VESC. open: No such file or directory failed ..
`
if i remove `<< port <<` it become 
`[vesc_driver]: Failed to connect to the VESC, SerialException Failed to open the serial port to the VESC. open: No such file or directory failed ..`

you are sure it is okay to delete it?